cms/djangoapps/course_creators/admin.py

```diff
@@ -30,26 +33,59 @@ def get_email(obj):
 get_email.short_description = 'email'
 
 
+class CourseCreatorForm(forms.ModelForm):
```

Comment:
Could you include a screenshot of the modified admin page, as well as a screenshot of every possible new view or error message that a Studio user might see? I'm going to have a my product manager give this a look-over too.

Reply:
Here are the set of screenshots for all errors


**When all_org is not selected and no org is added**

![image](https://user-images.githubusercontent.com/7670449/128209710-11b91724-e973-400c-9eb6-4488fbd25559.png)

**When all_org and specific org both are selected**

![image](https://user-images.githubusercontent.com/7670449/128210135-a13128ad-a60e-4763-b925-93db9e562956.png)

**Studio Error when we put wrong org name**

![image](https://user-images.githubusercontent.com/7670449/128211264-ff9e3265-c462-4653-a687-c922c831e409.png)

Reply:
@farhaanbukhsh Excellent. Can you move these to the PR description, and update description text as well? Try to separate the product change description from the engineering implementation details and testing instructions.

I will be showing this to my product manager soon. The clearer it is to her what the change is and how it'll affect the Studio product, the quicker we can get this review wrapped up and the PR merged 😃

Reply:
@kdmccormick  Thanks a bunch for looking at this 😄 . I have added a `product change description` in the description and have tried to explain the changes from a `product` POV. Please let me know if it's okay or should we make some changes.

Hope this PR gets to completions soon. 🤞🏾

Reply:
@farhaanbukhsh Perfect, thank you.